packages/stats/gbstats/bayesian/tests.py

```diff
@@ -38,6 +41,19 @@ class EffectBayesianConfig(BayesianConfig):
     prior_effect: GaussianPrior = field(default_factory=GaussianPrior)
 
 
+@dataclass
+class BanditConfig(BayesianConfig):
+    bandit_weights_seed: int = 1
```

Comment:
nit: IMO a seed shouldn't have a fixed default. It should be unset perhaps as a fallback.

Reply:
changed default seed to be 0.  changed `compute_variation_weights` so the seed is used only if positive, so unset is the default.